webgl-renderers/WebGLRenderer.js

```diff
@@ -777,14 +912,19 @@ WebGLRenderer.prototype.drawBuffers = function drawBuffers(vertexBuffers, mode,
  */
 WebGLRenderer.prototype.updateSize = function updateSize(size) {
     if (size) {
```

Comment:
Why would you call `.updateSize` and not pass in a size? This seems weird to me.